### PR TITLE
Url conversion

### LINKED ISSUE #4: Update Nodejs client to be able to send conversion urls to server.
Update the client in relation to issue 6 on base server repo. see [issue 6](https://github.com/idrsolutions/base-microservice-example/issues/6)

### PR DESCRIPTION
**Pull request by patricksheppard-IDR**
Fixes #4 

This PR does not break backwards compatibility so I have only updated the minor version number.

### CODE REVIEW

File: buildvu-client.js [hunk 1]
```diff
@@ -89,35 +90,46 @@ var fs = require('fs');
...
+                    } else {
+                        throw Error('Did not recognise type of file');
+                    }
                 } else {
-                    throw Error('Did not recognise type of file');
+                    isUrlInput = true;
                 }
+            
 
                 var formData = params.parameters || {};
```
Comment (leonatherton) [inline buildvu-client.js]: Can we remove the isUrlInput boolean if we move the formData initialization to before line 93?

File: buildvu-client.js [hunk 2]
```diff
@@ -89,35 +90,46 @@ var fs = require('fs');
...
+                if (params.filename) {
+                    formData.filename = params.filename;
+                }
                 var options = {
                     method: 'POST',
                     uri: params.endpoint,
-                    formData: formData
-                };
+                    formData: formData,
+                }
```
Comment (leonatherton) [inline buildvu-client.js]: Deliberately or accidentally changed?

File: buildvu-client.js (hunk 1 above)
Comment (patricksheppard-IDR) [reply buildvu-client.js]: Have changed it, good catch

File: buildvu-client.js (hunk 2 above)
Comment (patricksheppard-IDR) [reply buildvu-client.js]: That was a mistake, have changed it now

**Pull request merged**